samples/nrf9160/mqtt_simple_only_secure/src/main.c

```diff
@@ -0,0 +1,434 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <nrf.h>
+#include <bsd.h>
+#include <lte_lc.h>
+#include <at_cmd.h>
+#include <at_notif.h>
+#include <net/mqtt.h>
+#include <net/socket.h>
+#include <net/bsdlib.h>
+#include <bsd_platform.h>
+
+static u8_t __attribute__((section (".bsdlib"), unused)) bsd_mem[BSD_RESERVED_MEMORY_SIZE];
```

Comment:
This is declared above as well. Is it meant to be declared twice?

Reply:
No, forgot to remove this when I moved the other into the bsdlib source files.